tad/core/db.py

```diff
@@ -44,20 +57,48 @@ def init_db():
     with Session(get_engine()) as session:
         if get_settings().ENVIRONMENT == "demo":
             logger.info("Creating demo data")
-
-            user = session.exec(select(User).where(User.name == "Robbert")).first()
-            if not user:
-                user = User(name="Robbert", avatar=None)
-                session.add(user)
-
-            status = session.exec(select(Status).where(Status.name == "Todo")).first()
-            if not status:
-                status = Status(name="Todo", sort_order=1)
-                session.add(status)
-
-            task = session.exec(select(Task).where(Task.title == "First task")).first()
-            if not task:
-                task = Task(title="First task", description="This is the first task", sort_order=1, status_id=status.id)
-                session.add(task)
-            session.commit()
+            remove_old_demo_objects(session)
```

Comment:
The alembic database script creates 4 statuses. These are not 'demo related' I think. So truncating would delete those. The 'remove old objects' deletes only the objects created by the previous demo function. So I think this would be the best way.

Reply:
I agree